packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/TableDemo/TableCollapsibleDemo.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+  SortByDirection,
+  headerCol,
+  TableVariant,
+  expandable,
+  cellWidth,
+  IRow,
+} from '@patternfly/react-table';
+
+export class TableCollapsibleDemo extends React.Component<{}, { columns: any, rows: IRow[] }> {
+  constructor(props) {
```

Comment:
Set props to be of type `TableProps`

Reply:
yes... this should help find any more issues 👍